phpseclib/Crypt/Base.php

```diff
@@ -873,16 +850,20 @@ function encrypt($plaintext)
      * If strlen($ciphertext) is not a multiple of the block size, null bytes will be added to the end of the string until
      * it is.
      *
-     * Note: Could, but not must, extend by the child Crypt_* class
-     *
      * @see Crypt_Base::encrypt()
      * @access public
      * @param String $ciphertext
      * @return String $plaintext
+     * @internal Could, but not must, extend by the child Crypt_* class
      */
     function decrypt($ciphertext)
     {
-        if ($this->engine == CRYPT_MODE_MCRYPT) {
+        if ($this->paddable) {
```

Comment:
Not sure how this change is related to "add setPreferredEngine". Seems to be unrelated cleanup.